include/votca/xtp/bse.h

```diff
@@ -62,6 +62,7 @@ class BSE {
     Index davidson_maxiter = 50;
     double min_print_weight =
         0.5;  // minimium contribution for state to print it
+    bool use_Hqp_offdiag = false;
```

Comment:
What does this line do?

Reply:
okay gwbse.cc has no tests so far, but then please remove this line.

Reply:
And where else is it supposed to go instead?

Reply:
where do you use it?

Reply:
I cannot find any place it is needed.

Reply:
In gwbse.cc only. Maybe it is better instead of changing the call to bse.configure in gwbse.cc depending on the option, to have the ignoring/including of the offdiags in the configure function, by

```suggestion
if ( _opt.use_Hqp_offdiag ){
_Hqp = AdjustHqpSize(Hqp_in, RPAInputEnergies);
} else {
_Hqp = AdjustHqpSize(Hqp_in, RPAInputEnergies).diagonal().asDiagonal();
}
```

Reply:
Different files, so the change is to be in bse.cc

Reply:
yes that is a better approach. Good idea.